tidy3d/web/cli/converter.py

```diff
@@ -0,0 +1,1340 @@
+            planewv += 1
+        elif line[:8] == "addtfsf;":
+            addtfsf_string, tfsf_name = _addtfsf(Lines, i + 1, tfsf)
+            tidy3d_file += addtfsf_string
+            sources += tfsf_name + ", "
+            tfsf += 1
+        elif line[:8] == "addmode;":
+            addmode_string, mode_src_name = _addmode(Lines, i + 1, modesrc)
+            tidy3d_file += addmode_string
+            sources += mode_src_name + ", "
+            modesrc += 1
+        elif line[:8] == "addfdtd;":
+            addfdtd_string, gridspec_string = _addfdtd(Lines, i + 1, variable_dict)
+            tidy3d_file += addfdtd_string
+        elif line[:8] == "addmesh;":
+            override_string, refine_box_name = _addmesh(Lines, i + 1, overstrct)
+            override_structures += refine_box_name + ", "
+            tidy3d_file += refine_box_name + override_string
+            overstrct += 1
+        elif line[:9] == "addpower;":
+            addpower_string, power_name = _addpower(Lines, i + 1, fluxmon)
+            monitors += power_name + ", "
+            tidy3d_file += power_name + addpower_string
+            fluxmon += 1
+        elif line[:17] == "addefieldmonitor;":
+            addefieldmonitor_string, monitor_name = _addefieldmonitor(Lines, i + 1, fieldmon)
+            monitors += monitor_name + ", "
+            tidy3d_file += monitor_name + addefieldmonitor_string
+            fieldmon += 1
+        elif line[:17] == "addmodeexpansion;":
+            addmodeexpansion_string, modemonitor_name = _addmodeexpansion(Lines, i + 1, modemon)
+            # not adding name first as we needed to create mode_spec
+            tidy3d_file += addmodeexpansion_string
+            monitors += modemonitor_name + ", "
+            modemon += 1
+        elif line[:5] == "set('":
+            pass
+        elif line[:14] == "switchtolayout":
+            pass
+        elif line[:9] == "selectall":
+            pass
+        elif line[:6] == "delete":
+            pass
+        else:
+            tidy3d_file += "# " + line[:-1] + " # does not yet parse to Tidy3d\n"
+    structures = structures[:-2] + "]"
+    sources = sources[:-2] + "]"
+    monitors = monitors[:-2] + "]"
+    override_structures = override_structures[:-2] + "]"
+
+    tidy3d_file += "\nsim = sim.copy(update=dict(structures=" + structures + ","
+    tidy3d_file += " # NOTE: Check order of structures for potential overlap issues\n"
+    tidy3d_file += "\tsources=" + sources + ",\n"
+    tidy3d_file += "\tmonitors=" + monitors + ",\n"
+    tidy3d_file += (
+        "\tgrid_spec="
+        + gridspec_string[:-1]
+        + ", override_structures="
+        + override_structures
+        + ")\n\t)\n"
+    )
+    tidy3d_file += ")"
+
+    return tidy3d_file
+
+
+# new_file_name = "new_tidy3d_file.py"
+# file_to_read = "example.lsf"
+
+
+def converter_arg(lsf_file, new_file):
+    if ".lsf" not in lsf_file:
+        print("Error:", lsf_file, "must be an .lsf file.")
```

Comment:
instead of this, we can just `raise Exception(f"{lsf_file} must be an .lsf file." There's no need to print and return. we can let python handle the error for us.

Note, this will also end the function execution, so no need to return.

Same for the out file handling below.

Both of these can just be regular if statements

```py
if ...:
    raise ...
if ...:
    raise ...
file_string = lsf_reader(...)

```